ci/it/testcases/utils.go

```diff
@@ -34,19 +36,53 @@ type Containers struct {
 	ClickHouse    *testcontainers.Container
 }
 
-func (c *Containers) Cleanup(ctx context.Context) {
-	if c.Elasticsearch != nil {
-		(*c.Elasticsearch).Terminate(ctx)
+func printContainerLogs(ctx context.Context, container *testcontainers.Container, name string) {
+	if container == nil {
+		return
 	}
-	if c.Quesma != nil {
-		(*c.Quesma).Terminate(ctx)
+
+	reader, err := (*container).Logs(ctx)
+	if err != nil {
+		log.Printf("Failed to get logs for container '%s': %v", name, err)
+		return
+	}
+
+	output, err := io.ReadAll(reader)
+	if err != nil {
+		log.Printf("Failed to read logs for container '%s': %v", name, err)
+		return
+	}
+
+	log.Printf("Logs for container '%s':", name)
+	for _, line := range strings.Split(string(output), "\n") {
+		log.Printf("Container '%s': %s", name, line)
```

Comment:
I would consider a more compact version, such as
```suggestion
		log.Printf("[%s]: %s", name, line)
```